content/intro-to-storybook/vue/fr/simple-component.md

```diff
@@ -140,7 +140,7 @@ Une autre bonne chose à propos du regroupement des `actionsData` dont un compos
 Lors de la création d'une story, nous utilisons une tâche de base (`taskData`) pour construire la forme de la tâche attendue par le composant. C'est généralement modelisé à partir de réelles données. Encore une fois, l'`export`ation de cette forme nous permettra de la réutiliser dans d'autres stories comme nous le verrons.
 
 <div class="aside">
-Les <a href="https://storybook.js.org/docs/vue/essentials/actions"><b>actions</b></a> vous aident à vérifier les interactions lors de la création des composants d'interface utilisateur de manière isolée. Souvent, vous n'aurez pas accès aux fonctions et aux états que vous dans le contexte de l'application. Utilisez <code>action()</code> pour les insérer.
+Les <a href="https://storybook.js.org/docs/vue/essentials/actions"><b>ns</b></a> vous aident à vérifier les interactions lors de la création des composants d'interface utilisateur de manière isolée. Souvent, vous n'aurez pas accès aux fonctions et aux états que vous dans le contexte de l'application. Utilisez <code>action()</code> pour les insérer.
```

Comment:
@scenaristeur, one quick note about this change if you don't mind. This sentence should actually be:

```md
<div class="aside">
<a href="https://storybook.js.org/docs/react/essentials/actions"><b>Actions</b></a> vous aident à vérifier les interactions lors de la construction des composants de l'UI en isolation. Souvent, vous n'aurez pas accès aux fonctions et à l'état dont vous disposez dans le contexte de l'application. Utilisez <code>action()</code> pour les simuler.
</div>
```
Instead of :
```md
div class="aside">
Les <a href="https://storybook.js.org/docs/vue/essentials/actions"><b>ns</b></a> vous aident à vérifier les interactions lors de la création des composants d'interface utilisateur de manière isolée. Souvent, vous n'aurez pas accès aux fonctions et aux états que vous dans le contexte de l'application. Utilisez <code>action()</code> pour les insérer.
</div>
```

We're referring to a Storybook addon which should be capitalized. If you're able to adjust it I'd appreciate it.